apps/db_dump/Makefile

```diff
@@ -6,24 +6,22 @@ CFLAGS_FINAL=${CFLAGS} ${BROKER_CFLAGS} ${BROKER_CPPFLAGS} -I${R}/include -I${R}
 OBJS = \
 	   db_dump.o \
 	   print.o \
-	   stubs.o
-
-BROKER_OBJS = \
-	   ${R}/src/memory_mosq.o \
-	   ${R}/src/memory_public.o \
-	   ${R}/src/packet_datatypes.o \
-	   ${R}/src/persist_read.o \
-	   ${R}/src/persist_read_v234.o \
-	   ${R}/src/persist_read_v5.o \
-	   ${R}/src/property_mosq.o \
-	   ${R}/src/topic_tok.o \
-	   ${R}/src/utf8_mosq.o
+	   stubs.o \
+	   memory_mosq.o \
+	   memory_public.o \
+	   packet_datatypes.o \
+	   persist_read.o \
+	   persist_read_v234.o \
+	   persist_read_v5.o \
+	   property_mosq.o \
+	   topic_tok.o \
+	   utf8_mosq.o
```

Comment:
I don't want to do this, I want to keep using the files as compiled for the broker.

Reply:
Ok. Simply dropped this commit.